.hound.yml

```diff
@@ -0,0 +1,4 @@
+eslint:
```

Comment:
This file should be communicating to the hound CI I cant see the houndCI doing anything